iosHyperskillApp/iosHyperskillApp/Sources/Modules/TopicToDiscoverNextCard/TopicToDiscoverNextCardDelegate.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+protocol TopicToDiscoverNextCardDelegate: AnyObject {
```

Comment:
Just for your information, we are following Apple's naming for Delegate's
https://vyahhi.myjetbrains.com/youtrack/articles/ALTAPPS-A-30/Swift-Style-Guide#delegaty

Reply:
and we are using `do` prefix when we call ViewModel directly `viewModel.doLoadContent()`

Reply:
Also, in the SwiftUI world delegate pattern is not common, but the delegate pattern is commonly used in the UIKit. So when we are working with SwiftUI -> callbacks are preferred, when working with UIKit -> delegate pattern is preferred.

Reply:
@vladkash for now it's OK (**not necessary to update the code**), but in the future please use info from the comments above.